spyder/api/widgets/__init__.py

```diff
@@ -766,7 +766,11 @@ def get_options_menu(self):
 
     def get_options_menu_button(self):
         """
+<<<<<<< HEAD
         Return the main options button of the widget.
+=======
+        Return the main options menu button of the widget.
+>>>>>>> Migrate Pylint plugin to new API
```

Comment:
@goapeca seems like the rebase had some issues, please fix this

Reply:
Oops. Fixing

Reply:
Fixed @dalthviz